circuit-types/src/match.rs

```diff
@@ -36,19 +33,12 @@ pub struct MatchResult {
     /// sells the quote
     pub direction: bool,
 
-    /// The following are supporting variables, derivable from the above, but
-    /// useful for shrinking the size of the zero knowledge circuit. As
-    /// well, they are computed during the course of the MPC, so it incurs
-    /// no extra cost to include them in the witness
-
-    /// The minimum amount of the two orders minus the maximum amount of the two
-    /// orders. We include it here to tame some of the non-linearity of the
-    /// zk circuit, i.e. we can shortcut some of the computation and
-    /// implicitly constrain the match result with this extra value
-    pub max_minus_min_amount: Amount,
```

Comment:
this field was a redundant artifact of how we used to structure this circuit, no longer needed so i removed it